frontend/src/pages/modelServing/screens/projects/NIMServiceModal/DeployNIMServiceModal.tsx

```diff
@@ -185,7 +215,7 @@ const DeployNIMServiceModal: React.FC<DeployNIMServiceModalProps> = ({
       editInfo?.inferenceServiceEditInfo?.spec.predictor.model?.runtime ||
       translateDisplayNameForK8s(createDataInferenceService.name, { safeK8sPrefix: 'nim-' });
 
-    const nimPVCName = getUniqueId('nim-pvc');
+    const nimPVCName = !editInfo ? getUniqueId('nim-pvc') : existingPVC?.metadata.name || 'nim-pvc';
```

Comment:
This is a good way to solve the lint error -- but I am not sure this is a good way to solve it logically. 

If for some reason you cannot get the PVC name, you should probably report an error and stop the submit. Otherwise you're just going to attach yourself to a different PVC and have issues.

---
Taking a step back and looking at this from the wider angle. 

I wonder if we can break this flow up a bit more and you just "keep existing data" in edit instead of trying to rebind the PVC name -- just skip the step, it's already bound to the existing inference service.

Reply:
Updated